Makefile

```diff
@@ -228,7 +233,7 @@ spellcheck-format-spelling:
 ci-install-minikube:
 	curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
 	sudo dpkg -i minikube_latest_amd64.deb
-	minikube start --vm-driver=docker --container-runtime=containerd --kubernetes-version=${KUBERNETES_VERSION}
+	minikube start --cpus='max' --memory='max' --vm-driver=docker --container-runtime=containerd --kubernetes-version=${KUBERNETES_VERSION}
```

Comment:
... increasing the CI VM size without increasing the minikube size... And I was wondering why resources were not used...